### ISSUE TITLE
Tables, migrations, controller

### ISSUE THREAD

**Issue by tigrasti** [maddhatter/laravel-fullcalendar#31]
Hi,

I'm trying all they to make this callendar work, but i cant do that. First, what tables or migration i have to make for this module? 

At the end of the process in read me file, i get this issue: ErrorException in 5533aaacf4ee07b4096d657b48e1cecb line 66:
Undefined variable: calendar,

In the controller, i get error - Undifined class Calendar. I try all namespaces and use, but nothing worked for me.

Thanks for help, 
Best regards

**Comment by Arios509** [comment]
Yeah, i not sure why this doesnt show up the view and link with my database

**Comment by ArtbyArjan** [comment]
Tables or migrations aren't necessary unless you want to pull data from a database.
If you do, you can create the migration and table in any way you want, you just have to make sure to provide the right information to the Calendar class.

There are two ways of using the `Calendar` class:
- with namespacing by placing the following rule on top of your controller:
`use MaddHatter\LaravelFullcalendar\Facades\Calendar;`
- without namespacing by placing a `\` in front of `Calendar`

Unfortunatly I can't show you any of my code as an example.
